apache/couchdb-fauxton
# Update documents/changes to use redux

## Overview

Update documents/changes to use redux

## Testing recommendations

- Go to Database > Changes page
- Verify list of changes is displayed
- Add a couple of filters. Verify only changes that match both filters are listed.
- Remove one of the filters. Verify changes listed match the filter.

## Checklist
